### PR TITLE
Github Sample Update

### PR DESCRIPTION
**Pull request by Bec-Lyons**
Update GitHub DevOps Sample

### CODE REVIEW

File: github-sample/.github/workflows/logicapp_deploy.yml [hunk 1]
```diff
@@ -4,67 +4,82 @@
...
+  logic_app_build: 
+    name: Build 
     runs-on: ubuntu-latest
     steps: 
-
     - name: Checkout
       uses: actions/checkout@v1
-    
+    - name: Package logic
+      run: (cd logic; mkdir -p ../build-art; zip -r ../build-art/logic.zip .)
```
Comment (derek1ee) [inline github-sample/.github/workflows/logicapp_deploy.yml:31]: With this change one don't need to specify name of every workflow in this action like before? That's great.

File: github-sample/.github/workflows/logicapp_deploy.yml [hunk 2]
```diff
@@ -4,67 +4,82 @@
...
-      id: fa
+    - name: Swap parameter files 
+      id: params
+      uses: azure/powershell@v1
       with:
-        app-name: ${{env.RG_LA}}
-        package: '${{ github.run_id }}.zip'
-        publish-profile: ${{steps.fncapp.outputs.profile}}
+        inlineScript: |
+          az functionapp deploy --resource-group ${{ secrets.RG_LA }} --name ${{ env.LA_NAME }} --src-path  logic/azure.parameters.json --type static --target-path parameters.json
```
Comment (derek1ee) [inline github-sample/.github/workflows/logicapp_deploy.yml:83]: is the logicapp version of the cli not available to do this?

File: github-sample/.github/workflows/logicapp_deploy.yml (hunk 1 above)
Comment (derek1ee) [reply github-sample/.github/workflows/logicapp_deploy.yml:31]: Should `build-art` be called `output` or something like that?

File: github-sample/ARM/la-parameters.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
     "contentVersion": "1.0.0.0",
     "parameters": { 
         "location": {
-            "value": "Australia Southeast"
+            "value": "australiasoutheast"
```
Comment (derek1ee) [inline github-sample/ARM/la-parameters.json:6]: maybe default to west us2?

**Review by derek1ee — APPROVED** [review]

File: github-sample/.github/workflows/logicapp_deploy.yml (hunk 2 above)
Comment (Bec-Lyons) [reply github-sample/.github/workflows/logicapp_deploy.yml:83]: not for this command

File: github-sample/ARM/la-parameters.json (hunk 3 above)
Comment (Bec-Lyons) [reply github-sample/ARM/la-parameters.json]: changed :)

**Pull request merged**